### DISCUSSION TITLE
PayPal Fenster für PRO Version verweist nur auf "Kreditkarte hinzufügen"

### DISCUSSION THREAD

**Discussion by ElAmarok** [Zefau/ioBroker.jarvis#2110] [General]
Guten Tag, ich weiß nicht ob das hier unter Diskussionen richtig ist.
Wenn ich die PRO Version über den Button in Jarvis kaufen will, öffnet sich das Fenster mit meinem PayPal Login und nach dem Login erscheint nur das Fenster für "Kreditkarte hinzufügen" obwohl ich per Banküberweisung genug Guthaben habe.
Mache ich da doch was falsch? Oder liegt das an meinem Paypal Account?

MfG
Patrick
![jarv1](https://user-images.githubusercontent.com/33393715/208322565-4284dac1-36e2-41b8-8315-685f25712752.PNG)

**Comment by Zefau** [comment]
Also ich habe nichts besonderes eingestellt, dass das Anlegen einer Kreditkarte erzwingt.

**Comment by ElAmarok** [comment]
Dachte ich mir schon. Ich hab es jetzt mal mit Überweisung versucht.
Danke.
